go.mod

```diff
@@ -61,7 +61,7 @@ require (
 	github.com/optiopay/kafka v0.0.0-00010101000000-000000000000
 	github.com/pborman/uuid v1.2.0
 	github.com/pmezard/go-difflib v1.0.0
-	github.com/prometheus/client_golang v1.7.1
+	github.com/prometheus/client_golang v1.11.0
```

Comment:
Looks good, we can't use v1.12 because it introduces new metrics which fail the metrics lint.